lightning/src/ln/onchaintx.rs

```diff
@@ -93,15 +93,62 @@ impl Readable for ClaimTxBumpMaterial {
 	}
 }
 
-#[derive(PartialEq)]
-pub(super) enum InputDescriptors {
+#[derive(PartialEq, Clone, Copy)]
+pub(crate) enum InputDescriptors {
```

Comment:
Should we be dropping the "s"? This doesn't seem to be plural, which I find confusing.

Reply:
Planning to move InputDescriptors to better-documented/generic TxTemplate as described in #606